src/data_snack/connections/mongo.py

```diff
@@ -20,53 +28,64 @@ def _get_entity_collection(self, entity_type: Type[Entity]) -> pymongo.collectio
         )
         return collection
 
-    def get(self, entity_type: Type[Entity], key: str) -> Optional[bytes]:
-        collection = self._get_entity_collection(entity_type)
-        result = collection.find_one({"_id": key})
+    def get(self, key: Key) -> Optional[bytes]:
```

Comment:
Mongo does not return bytes, but a Dict.

Reply:
Yes, I left the old typing, fixed.